### PR TITLE
Feature/crude styling

### LINKED ISSUE #1: Header component
- [x] Add crude jsx
- [x] Add crude styling
- [x] Test

### PR DESCRIPTION
**Pull request by leighlars**
# Description
Adds Header feature-- logo and search form + crude styling and testing
Font, colors, layout can all change. Might as well look at something pretty as we code along!

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality) #1 
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [X] Yes

I tested `Search` to check if form was rendered and the value was changed on `fireEvent`. I tested `Header` and `App` to make sure elements were rendered on each JSX element. Run `npm test` to view. 

# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Header/Header.test.tsx [hunk 1]
```diff
@@ -7,8 +7,16 @@ describe('Header', () => {
     render(<Header/>)
   })
 
-  it('should render app name in Header', () => {
-    const header = screen.getByRole('heading', {name: 'Weather Or Not'})
-    expect(header).toBeInTheDocument()
+  it('should render logo and search bar in Header', () => {
```
Comment (JoshSevy) [inline src/Header/Header.test.tsx:10]: Awesome work. Getting a head start on that testing love it.

File: public/index.html [hunk 2]
```diff
@@ -2,14 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/globe.png" />
```
Comment (ErinUntermeyer) [inline public/index.html:5]: Great addition, it feels so official now!

File: src/Header/Header.test.tsx (hunk 1 above)
Comment (leighlars) [reply src/Header/Header.test.tsx:10]: As Ian said in PD today, 'what's a weakness you won't bring _here_?"

File: src/Header/Header.tsx [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
 import Search from '../Search/Search'
+import './Header.scss'
+import globe from '../assets/globe.png'
 
 const Header: React.FC = () => {
   
   return (
-    <article className="header-container">
+    <header className="header-container">
+      <div className='logo-box'>
```
Comment (JoshSevy) [inline src/Header/Header.tsx:10]: The logo looks great. Thanks for getting some styling done.

File: src/Search/Search.test.tsx [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+  beforeEach(() => {
+    render(<Search />)
+  })
+
+  it('should render a search bar', () => {
+    const search = screen.getByPlaceholderText('Search city, zip, or county')
+    expect(search).toBeInTheDocument()
+  })
+
+  it("should reflect change in value when data is input in form", () => {
```
Comment (ErinUntermeyer) [inline src/Search/Search.test.tsx:15]: Thanks for getting these tests done to ensure the search bar is working properly!

File: src/styles/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@700&display=swap');
+$darkBlue: #013578;
+$lightBlue: #33a6f2;
+
+
+@mixin rowDisplay {
```
Comment (JoshSevy) [inline src/styles/_variables.scss:6]: Awesome to see we are already starting to make some mixins. Great thinking and code looks great!

**Pull request merged**